playbook/lib/playbook/kit_base.rb

```diff
@@ -78,8 +80,30 @@ def combined_html_options
       default_html_options.merge(html_options.deep_merge(data_attributes))
     end
 
+    # rubocop:disable Style/OptionalBooleanParameter
+    def pb_content_tag(name, content_or_options_with_block = nil, options = nil, escape = true, &block)
```

Comment:
Not sure that this will save any code on our part as the helper is, by default, using positional arguments instead of named params. This was done to keep things as familiar to `content_tag` as possible.